stix_shifter_modules/config.json

```diff
@@ -11,7 +11,7 @@
                 "optional": true
             },
             "result_limit": {
-                "default": 10000,
+                "default": 1000,
```

Comment:
why did you change the default value in the common config file?

Reply:
if CrowdStrike api has a different default result limit than it should be specified in `CrowdStrike/configuration/config.json`